CHANGELOG.md

```diff
@@ -4,6 +4,37 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 ## [Prerelease] - Unreleased
 
+## [v0.5.0] - 2025-05-28
+
+### Changed
+* Change base address from 0x200_000 to 0x0 by @danbugs in https://github.com/hyperlight-dev/hyperlight/pull/450
+* Unify HostFunctionXX traits into a single HostFunction by @jprendes in https://github.com/hyperlight-dev/hyperlight/pull/
+* Improve the ergonomics of registering host functions by @jprendes in https://github.com/hyperlight-dev/hyperlight/pull/468
+* Remove generics from SupportedParameterType and SupportedReturnType traits by @jprendes in https://github.com/hyperlight-dev/hyperlight/pull/475
+* Improve ergonomics of SupportedParameterType and SupportedReturnType by @jprendes in https://github.com/hyperlight-dev/hyperlight/pull/476
+* Remove `host_print_writer` from the arguments to `UninitializedSandbox::new` by @jprendes in https://github.com/hyperlight-dev/hyperlight/pull/487
+
+### Fixed
+* Add error logging when MapViewOfFileNuma2 fails by @ludfjig in https://github.com/hyperlight-dev/hyperlight/pull/460
+* Make common and guest libs portable by @danbugs in https://github.com/hyperlight-dev/hyperlight/pull/524
+* Fix breaking changes for hyperlight js by @simongdavies in https://github.com/hyperlight-dev/hyperlight/pull/531
+
+### Added
+* Gdb debug improvements by @dblnz in https://github.com/hyperlight-dev/hyperlight/pull/456
+
+### Removed  
+* Remove kernel stack and boot stack memory regions by @danbugs in https://github.com/hyperlight-dev/hyperlight/pull/451
+* Removing HostFunctionDefinitions region by @danbugs in https://github.com/hyperlight-dev/hyperlight/pull/453
+* Removed host error region by @danbugs in https://github.com/hyperlight-dev/hyperlight/pull/457
+* Remove dependency on the paste crate by @jprendes in https://github.com/hyperlight-dev/hyperlight/pull/467
+* Remove support from hyperlight_host for PE formatted guests by @simongdavies in https://github.com/hyperlight-dev/hyperlight/pull/485
+* Remove in process mode from hyperlight-host by @simongdavies in https://github.com/hyperlight-dev/hyperlight/pull/490
+
+## New Contributors
```

Comment:
The "new contributors" is added automatically, along with the "full changelog", so you shouldn't need to add them manually. You can see exactly what it will look like locally by running `just create-release-notes v0.5.0 > notes.md`

Reply:
Should be fixed by https://github.com/hyperlight-dev/hyperlight/pull/539, but that shouldn't block this PR